frontend/packages/core/src/Contexts/wizard-context.tsx

```diff
@@ -1,8 +1,14 @@
 import React from "react";
 
+export interface NavigationProps {
```

Comment:
Lets rename this to `WizardNavigationProps` or adjust the export in the index to rename it, just so it doesn't get confused with anything (since we do have a navigation setup in Core already)